src/client/pps.go

```diff
@@ -681,11 +681,11 @@ func (c APIClient) CreatePipeline(
 }
 
 // InspectPipeline returns info about a specific pipeline.
-func (c APIClient) InspectPipeline(pipelineName string, details bool) (*pps.PipelineInfo, error) {
+func (c APIClient) InspectPipeline(pipelineAncestry string, details bool) (*pps.PipelineInfo, error) {
```

Comment:
The ancestry syntax should be resolved as close to the gRPC boundary (or more generally the boundary where we need it resolved) as possible to prevent that rather than having to track them separately. This pattern makes sense generally for values that are valid in requests, but not internally. For example, we do this with path cleaning when working with files. This allows us to be more flexible with how users can interface with our API while not incurring additional complexity internally.

Reply:
Oh, are you saying that this should just take a plain name and not allow the ancestry syntax? That makes sense. I’ll see if I can make that change work.